notifique/src/production/res/layout/main.xml

```diff
@@ -17,7 +17,5 @@
       android:id="@+id/list"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:scrollbarThumbVertical="@drawable/scrollbar"
```

Comment:
make this match the internal layout resource.
im guessing you want production to have this feature, too.
and thats why youre seeing the unused resource failure.

Reply:
updated prod